src/main/java/com/atherys/towns/util/MathUtils.java

```diff
@@ -1,15 +1,39 @@
 package com.atherys.towns.util;
 
+import com.atherys.towns.model.entity.Plot;
 import com.flowpowered.math.vector.Vector2i;
 import com.flowpowered.math.vector.Vector3d;
 import com.flowpowered.math.vector.Vector3i;
+import org.spongepowered.api.entity.living.player.Player;
```

Comment:
Not used, but we shouldn't be using any MC class in here as we would like to write Unit tests for this.

Reply:
That's completely fair, I will change that.